.github/workflows/ci-reproducibility.yml

```diff
@@ -0,0 +1,38 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-reproducibility
+
+# Trigger the workflow when:
+on:
+  push:
+    branches:
+      - main
+      - stable/*
+  # Besides pushes on the branches above, also check every day at 00:00 UTC.
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+
+  check-reproducible-build:
```

Comment:
Can you create two jobs, one for ELF binaries and another one for SGXS so that we test reproducibility of both.